Formula/guile-gnutls.rb

```diff
@@ -0,0 +1,51 @@
+class GuileGnutls < Formula
+  desc "Guile bindings for the GnuTLS library"
+  homepage "https://gitlab.com/gnutls/guile"
+  url "https://gitlab.com/gnutls/guile/uploads/56e18069ab63ca67d61aecb6b2697ec1/guile-gnutls-3.7.11.tar.gz"
+  sha256 "058eaa5c763e19fbf93e8b4eefc11280f8070535c138c99be11f00cd685613df"
+  license "LGPL-2.1-or-later"
+
+  depends_on "gnutls"
+  depends_on "guile"
+
+  def install
+    # configure: WARNING: unrecognized options: --disable-debug
+    system "./configure", "--prefix=#{prefix}",
+                          "--with-guile-site-dir=#{share}/guile/site/3.0",
+                          "--with-guile-site-ccache-dir=#{lib}/guile/3.0/site-ccache",
+                          "--with-guile-extension-dir=#{lib}/guile/3.0/extensions",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules"
+    system "make", "install"
+  end
+
+  def post_install
+    # Touch gnutls.go to avoid Guile recompilation.
+    # See https://github.com/Homebrew/homebrew-core/pull/60307#discussion_r478917491
```

Comment:
would this trigger audit failure in the future?

Reply:
No